### PR TITLE
Feature: Nessus & CIS CAT Converters

### PR DESCRIPTION
**Pull request by ejohn20**
Creates a new converter + unit tests for Nessus (XML format) and CIS CAT (JSON format) covered in https://github.com/microsoft/sarif-sdk/issues/2531

### CODE REVIEW

File: src/Test.UnitTests.Sarif.Converters/TestData/NessusConverter/ExpectedOutputs/ValidResults.sarif [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{
```
Comment (yongyan-gh) [inline src/Test.UnitTests.Sarif.Converters/TestData/NessusConverter/ExpectedOutputs/ValidResults.sarif:1]: @ejohn20 can you please also share a sample Nessus Sarif file and a sample CIS CAT Sarif file with real world rules/results data, so team can review them on different viewers and provide feedbacks?

Comment (ejohn20) [reply src/Test.UnitTests.Sarif.Converters/TestData/NessusConverter/ExpectedOutputs/ValidResults.sarif:1]: @yongyan-gh - Yes I'll provide them to you via email. Thanks.

File: src/Sarif.Converters/CisCatConverter.cs [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            }
+
+            var results = new List<Result>();
+            foreach (CisCatRule rule in log.Rules)
+            {
+                if (!rule.IsPass())
+                {
+                    results.Add(CreateResult(rule));
+                }
+            }
```
[the remark this answers is not in the document]
Comment (ejohn20) [reply src/Sarif.Converters/CisCatConverter.cs]: Fixed in 366bdde

**Comment by ejohn20** [comment]
@yongyan-gh Sample sarif files provided via email to your profile. CodeQL + dotnet format errors resolved. Let me know if anything else needs to be done on this one.

**Pull request merged**